File: src/main/java/com/jetbrains/gymusserverjava/store/dtos/responses/ProductResponseDto.java
package com.jetbrains.gymusserverjava.store.dtos.responses;

import java.time.LocalDateTime;

public record ProductResponseDto(
        int id,
        String productName,
        String productImage,
        String productDescription,
        int quantity,
        double price,
        int addedBy,
        LocalDateTime createdAt,
        LocalDateTime updatedAt
) {}

File: src/main/java/com/jetbrains/gymusserverjava/store/dtos/responses/ProductResponseDto.kt
package com.jetbrains.gymusserverjava.store.dtos.responses

import java.time.LocalDateTime

@JvmRecord
data class ProductResponseDto(
    val id: Int,
    val productName: String,
    val productImage: String,
    val productDescription: String,
    val quantity: Int,
    val price: Double,
    val addedBy: Int,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime
)
